### ISSUE TITLE
Transparent background

### ISSUE THREAD

**Issue by luisdavim** [tomasr/molokai#50]
Is there a way to make the background transparent (use the terminal's background)?
set transparency=15 "only works in MacVim

**Comment by justinmk** [comment]
Not from Vim... that's a terminal feature.

**Comment by luisdavim** [comment]
Yes, but this color scheme overrides the terminal background with ctermbg.
What I'm asking is for some option to toggle that, something that I can add to my .vimrc like let g:monokai-transparent = 1

**Comment by justinmk** [comment]
The way to override colorscheme settings is to `:hi ctermbg=foo`. Not to add random settings.

**Comment by zergler** [comment]
This worked for me with a transparent rxvt terminal. Make sure to add it after your colorscheme molokai and you shouldn't have to modify the color file at all. If you want to toggle it within vim, wrap it in a vim function.

hi Normal ctermbg=none
hi NonText ctermbg=none
hi LineNr ctermbg=none

**Comment by guiniol** [comment]
I proposed a PR a few years back to the same effect: #41 
It adds an option: `g:molokai_transparent` which basically does `hi Normal ctermbg=none`

**Comment by jgrar** [comment]
Here's my example if you're interested jgrar/dotfiles@3204e17

There's an example screenshot at the bottom.

**Comment by selurvedu** [comment]
@jgrar, thanks, I'll give it a shot.
